deploy/overview.mdx

```diff
@@ -155,29 +155,36 @@ A Chainlit application can be consumed through multiple platforms. Write your as
 </Card>
 </CardGroup>
 
-## Deploy your Chainlit Application
+## Tips & Tricks
 
-No matter the platform(s) you want to serve with your Chainlit application, you will need to deploy it first.
+### Start Chainlit with -h
 
-After you've successfully set up and tested your Chainlit application locally, the next step is to make it accessible to a wider audience by deploying it to a hosting service. This guide provides various options for self-hosting your Chainlit app.
+When running a Chainlit app in production, you should always add `-h` to the
+`chainlit run` command. Otherwise a browser window will be opened server side
+and might break your deployment.
+
+### Account for websockets
+Chainlit is built upon websockets, which means the service you deploy your app
+to has to support them. When auto scaling, make sure to enable sticky sessions (or session affinity).
+
+Even with sticky sessions, load balancers sometime struggle to consistently route a client to the same container.
+In that case you can set `transports = ["websocket"]` in your `.chainlit/config.toml` file.
+
+### Deploying Chainlit on a subpath
+If you need to deploy your Chainlit app to a subpath like
+`https://my-app.com/chainlit`, you will need to set the `--root-path
+/chainlit` flag when running the `chainlit run` command. This will ensure that
+the app is served from the correct path.
 
-<Note>
-  When running a Chainlit app in production, you should always add `-h` to the
-  `chainlit run` command. Otherwise a browser window will be opened server side
-  and might break your deployment.
-</Note>
-
-<Note>
-  Chainlit is built upon websockets, which means the service you deploy your app
-  to has to support them. For auto scaling, make sure to enable sticky sessions.
-</Note>
-
-<Warning>
-  If you need to deploy your Chainlit app to a subpath like
-  `https://my-app.com/chainlit`, you will need to set the `--root-path
-  /chainlit` flag when running the `chainlit run` command. This will ensure that
-  the app is served from the correct path.
-</Warning>
+### Cross origins
+If your end users consumes the Chainlit UI from the same origin as the server, everything will work out of the box.
+However, if you embed Chainlit on a website, the connection will fail because of CORS.
```

Comment:
I think this is not clear enough. The only place the origin does not need to be set is if you're serving chainlit locally on your machine. In any other _actual_ deployment scenario it will need to be set.

Not everyone who might use chainlit can be expected to be a web developer, they might not know what an origin or CORS are. We don't need to fully explain that, we just need to tell them they need to set it to the address at which chainlit is served. 

I think the equivalent setting in Django explains it pretty well - it's just the host and port where chainlit is served/reachable from. https://docs.djangoproject.com/en/5.1/ref/settings/#allowed-hosts

In addition, an example here would mean the world - so users don't have to go somewhere else and can just follow this like a deploy playbook.